docs/schedule/speaking.md

```diff
@@ -9,6 +9,7 @@ PyTexas is a single-track conference with a total attendence of ~250 people.
 ## Call for proposals
 
 * The Call For Proposals (CFP) opens **October 1, 2024**
+* The Call for Proposals (CFP) closes **December 1, 2024**
```

Comment:
Do we also need to update the `description` at the top of the file?

Reply:
Good catch. Updated.